elements/pl-python-variable/pl-python-variable.py

```diff
@@ -0,0 +1,44 @@
+import prairielearn as pl
+import lxml.html
+
+NO_HIGHLIGHT_DEFAULT = False
+PREFIX_DEFAULT = ''
+SUFFIX_DEFAULT = ''
+FORCE_TEXT_DEFAULT = False
```

Comment:
We are trying to keep the `_DEFAULT` names exactly the same as `<attribute-name>_DEFAULT`, so remove `FORCE_`?